lib/src/ui/pages/authentication/login/login_notifier.dart

```diff
@@ -50,6 +50,7 @@ class LoginNotifier extends ChangeNotifier {
     _state = _state.copyWithFieldsErrors(
```

Comment:
jakbyś loading pokazywał tuż przed strzałem to nie trzeba by było tu chować progresu i zastanawiać się czy może jakoś ładniej nazwać tę metodę, bo ten loading psuje sygnaturę :D